neuralmonkey/learning_utils.py

```diff
@@ -247,7 +248,7 @@ def training_loop(tf_manager: TensorFlowManager,
                         _log_continuous_evaluation(
                             tb_writer, tf_manager, main_metric, val_evaluation,
                             seen_instances, epoch_n, epochs, val_results,
-                            train=False)
+                            train=False, dataset_name=valset.name)
```

Comment:
tak jak je to udělaný, tak bych tu poskytnul `valset.name` jen v případě, že je víc než jeden validační dataset.